README.md

```diff
@@ -2,23 +2,23 @@
 
 ###Project stats: [![Build Status](http://jenkins.autonomiccs.com.br/buildStatus/icon?job=Autonomiccs-platform)](http://jenkins.autonomiccs.com.br/job/Autonomiccs-platform/) <a href="https://scan.coverity.com/projects/autonomiccs-platform"><img alt="Coverity Scan Build Status" src="https://scan.coverity.com/projects/8610/badge.svg"/></a>
 
-Autonomiccs platform is an open source software that provides the vital components needed to manage a cloud computing environment autonomously; we act at the infrastructure as a service level by providing a plugin to <a href="https://github.com/apache/cloudstack">Apache CloudStack</a>.
+The Autonomiccs platform is an open source software that provides the vital components needed to manage a cloud computing environment autonomously; we act at the infrastructure as a service level by providing a plugin to <a href="https://github.com/apache/cloudstack">Apache CloudStack</a>.
 
-Our mission is to take the orchestration of cloud computing infrastructures to the next level. To achieve that, we developed a distributed and autonomic plugin that can monitor the CC environment and take decisions in order to fulfill the goals of the given environment. We do not wish to remove the need for human administrators, but to improve their work. Our mission is to optimize and make CC environments more efficient and stable without needing to rely on human administrators.
+Our mission is to take the orchestration of cloud computing infrastructures to the next level. To achieve that, we've developed a distributed and autonomic plugin that can monitor a CC environment and take decisions in order to fulfill the goals of said environment. We do not wish to remove the need for human administrators, but to improve their work. Our mission involves optimizing and making CC environments more efficient and stable without the need to rely on human administrators.

```

Comment:
@cristofolini here you use a contraction:

> we've
but in writing I think we should avoid them. Could you use the full version of the verb?
> we have